examples/v2/chat2.nim

```diff
@@ -78,8 +78,27 @@ proc initAddress(T: type MultiAddress, str: string): T =
 
 proc parsePeer(address: string): PeerInfo = 
```

Comment:
Does it have any unit test? if not, might be good to have it tested

Reply:
Agree! This is quite a central part of `wakunode2` connectivity. Will add.

Reply:
Done in 105bbd4.

Also extracted this functionality to a utils module. Cleaning up imports as I go along.